static/app/components/issues/groupList.tsx

```diff
@@ -35,8 +35,11 @@ const defaultProps = {
   useFilteredStats: true,
   useTintRow: true,
   narrowGroups: false,
+  withColumns: ['graph', 'event', 'users', 'assignee', 'lastTriggered'] as Column[],
 };
 
+export type Column = 'graph' | 'event' | 'users' | 'assignee' | 'lastTriggered';
```

Comment:
this type needs a better name if we're exporting it GroupListColumns or something